cpp/tests/filling/filling_tests.cu

```diff
@@ -35,13 +35,18 @@ TYPED_TEST_CASE(FillingTest, test_types);
 
 constexpr gdf_size_type column_size{1000};
 
+using column_validity_fn_t = std::function<bool(gdf_index_type)>;
+
+column_validity_fn_t all_valid = [](gdf_index_type row) { return true; };
+
 template <typename T>
 void FillTest(gdf_index_type begin, gdf_index_type end,
-              T value, bool value_is_valid = true)
+              T value, bool value_is_valid = true, 
+              column_validity_fn_t column_validity_fn = all_valid)
```

Comment:
This is kind of a clunky approach, especially using the same validity generator func for source and expected.  I think I would rather you just make the source column wrappers a parameter to FillTest, like here: https://github.com/rapidsai/cudf/blob/fa94941c8b61927a6fea7880a9dc322c986b9a20/cpp/tests/stream_compaction/stream_compaction_tests.cu#L118-L137

Reply:
Or if you go with this approach, make the functor type a template parameter called `typename BitGenerator` and change this line to:
```suggestion
              BitGenerator column_validity_fn = all_valid)
```